### DISCUSSION TITLE
Bitcoin wallet not open.

### DISCUSSION THREAD

**Discussion by Pavlos584** [btcpayserver/btcpayserver#5343] [Technical Support]
Hello, after the server update, I have problems accessing the wallets. After clicking no Bitcoin, it just shows a white page with nothing. I have version 1.11.4.

**Comment by Pavlos584** [comment]
![IMG_6101](https://github.com/btcpayserver/btcpayserver/assets/108590410/23ced8b7-f669-444c-912c-7358e47bc1d5)
![IMG_6100](https://github.com/btcpayserver/btcpayserver/assets/108590410/e95ed422-179b-45b2-a1e4-cc1e118a1381)

**Reply by Pavlos584** [reply]
I have run btcserver on 3 different instances and the problem is still the same.

**Comment by dennisreimann** [comment]
Will be fixed with the v1.11.5 we are about to release today.

**Comment by Markywa571** [comment]
Hi 🖐🏻
I have a suggestion on how to legalize your cryptocurrency💰
We can offer Bitcoin Mixer
Completely anonymous👀
No registration, no logs 
Available 24/7
Website http://f7kv6nuu6swk6cybbi3soqdvg7enu4gammot5yc3h2zbbimevufnbkad.onion/
